docker/nginx-default.conf

```diff
@@ -1,26 +1,19 @@
 server {
     listen 3001;
 
-    location /v2/debug/ {
```

Comment:
I'm going to upgrade to aeproject `3.x.x` as well, and this will be fixed. Working on the test right now, as the new aeproject has some changes in network initialization and contract code fetch from the filesystem :D